backend/service/topology/cache.go

```diff
@@ -66,3 +69,61 @@ func convertLockIdToAdvisoryLockId(lockID string) uint32 {
 	x := sha256.New().Sum([]byte(lockID))
 	return binary.BigEndian.Uint32(x)
 }
+
+// TODO (mcutalo): make this private once its in use
+func (c *client) SetCache(ctx context.Context, obj *topologyv1.Resource) error {
```

Comment:
can use lint directives instead

```suggestion
// nolint:unused
// TODO (mcutalo): remove lint directive once in use
func (c *client) setCache(ctx context.Context, obj *topologyv1.Resource) error {
```